src/common/reducers/handlers/cards.js

```diff
@@ -39,9 +41,22 @@ const cardsHandlers = {
     // Add new cards that are not duplicates.
     cardsFromJson(json)
       .filter(card => !state.cards.map(c => c.id).includes(card.id))
```

Comment:
I think we need to filter out `id` and `timestamp` here. But we also need to validate each field and not just trust the user.